doc/specs/#5000 - Process Model 2.0/#4472 - Windows Terminal Session Management.md

```diff
@@ -0,0 +1,465 @@
+---
+author: Mike Griese @zadjii-msft
+created on: 2020-10-30
+last updated: 2020-11-02
+issue id: #4472
+---
+
+# Windows Terminal Session Management
+
+## Abstract

```

Comment:
Pro tip: You need a newline below the title. Otherwise, Github will fail to render it correctly.

Reply:
What, they changed this behavior again? At first it was working. At some point around the year 2018 it stopped working. Apparently it's working again. Feel free to ignore this.